### PR TITLE
Add BPF target to LLVM

### PR DESCRIPTION
**Pull request by jpsamaroo**
@vchuravy

### CODE REVIEW

**Comment by vchuravy** [comment]
You also need to touch `LLVM@10.0.1/build_tarballs.jl`, by adding a newline.

File: L/LLVM/common.jl [hunk 1]
```diff
@@ -103,9 +103,9 @@ fi
 
 # build for our host arch and our GPU targets NVidia and AMD
 TARGETS=(host NVPTX AMDGPU)
-# Add WASM for LLVM 8
+# Add WASM and BPF for LLVM 8
```
Comment (giordano) [inline L/LLVM/common.jl:106]: This comment referring to LLVM 8 isn't very accurate anymore

**Comment by giordano** [comment]
@vchuravy good to merge I guess?

**Comment by vchuravy** [comment]
Julian, If you want it in master, you will have to go bump the build id and
update the checksums. Otherwise this will land eventually ;)

**Pull request merged**